app/app.go

```diff
@@ -195,6 +203,33 @@ func ProjectUpAndWait(p *project.Project, waiter options.Waiter, c *cli.Context)
 		return nil
 	}
 
+	if c.Bool("prune") {
+		url := c.GlobalString("url")
+		accessKey := c.GlobalString("access-key")
+		secretKey := c.GlobalString("secret-key")
+
+		apiClient, err := client.NewRancherClient(&client.ClientOpts{
```

Comment:
We shouldn't spin up a new client for this. There has to be a client that exists somewhere else. It should be reused